service/application/docker/Dockerfile

```diff
@@ -20,16 +20,19 @@ WORKDIR /opt/ema
 
 ARG PLATFORM=linux_amd64
 
-COPY tofu_1.7.0_amd64.apk /opt/ema/terraform
-RUN apk --update add --allow-untrusted /opt/ema/terraform/tofu_1.7.0_amd64.apk
+COPY tofu_1.8.2_amd64.apk /opt/ema/terraform
+RUN apk --update add --allow-untrusted /opt/ema/terraform/tofu_1.8.2_amd64.apk
 
 COPY .terraformrc $HOME/.terraformrc
 RUN printf '#!/bin/ash\ntofu $*' > /opt/ema/terraform/terraform
 RUN chmod +x /opt/ema/terraform/terraform
 ENV PATH $PATH:/opt/ema/terraform
 
-ARG SOLACE_PROVIDER_VERSION=0.9.2-rc.2
-ADD terraform-provider-solacebroker_${SOLACE_PROVIDER_VERSION}_${PLATFORM}.tar.gz /opt/ema/terraform
+ARG SOLACE_PROVIDER_VERSION=1.0.1
+ADD terraform-provider-solacebroker_${SOLACE_PROVIDER_VERSION}_${PLATFORM}.zip /opt/ema/terraform
+
```

Comment:
why zip file? 
you could use .tar and just use add (I think add also works with zip)

Reply:
Because this [version 1.0.1](https://github.com/SolaceProducts/terraform-provider-solacebroker/releases/tag/v1.0.1) only had the zip format available.

As for the command I followed the steps from [this confluence page](https://sol-jira.atlassian.net/wiki/spaces/EP/pages/3848569320/How+to+update+the+terraform+solace+provider+in+the+EMA#Steps). Please let me know if that's wrong instruction!

Reply:
The ADD command already extracts content no? 
the point I am trying to make is most likely, you do not need the unzip command https://docs.docker.com/build/building/best-practices/#add-or-copy

Reply:
I tested removing the unzip command:
<img width="1011" alt="Screenshot 2024-09-17 at 11 43 16 AM" src="https://github.com/user-attachments/assets/c6449c4b-3476-48d7-b3c0-32b23d27734f">

The result was that I will be missing the terraform-provider if I do that:
<img width="1027" alt="Screenshot 2024-09-17 at 11 18 17 AM" src="https://github.com/user-attachments/assets/9c2f8511-ed78-4acc-aeed-49694e5e0f12">
<img width="787" alt="Screenshot 2024-09-17 at 11 42 17 AM" src="https://github.com/user-attachments/assets/762b6e04-0397-49b1-ab0c-4d60a8890076">